config/jobs/kubernetes/minikube/minikube.yaml

```diff
@@ -0,0 +1,28 @@
+presets:
+- env:
+  - name: GCS_CREDS
+    value: /root/.config/k8s-minikube-20e2437ce844.json
+  volumes:
+  - name: k8s-minikube-build-gcs
+    secret:
+      secretName: k8s-minikube-build-gcs
+      items:
+      - key: k8s-minikube-20e2437ce844.json
+        path: k8s-minikube-20e2437ce844.json
```

Comment:
You don't need to specify the `items` section, the `path` defaults to the `key`.

Reply:
cool.  removed.